### ISSUE TITLE
[vizio] SSL warnings in openhab.log file

### ISSUE THREAD

**Issue by robnielsen** [openhab/openhab-addons#14687]
After installing the vizio binding, I am now seeing the following in the openhab.log file:
```
2023-03-25 20:00:34.288 [WARN ] [ty.util.ssl.SslContextFactory.config] - Trusting all certificates configured for Client@24875bf8[provider=null,keyStore=null,trustStore=null
]
2023-03-25 20:00:34.291 [WARN ] [ty.util.ssl.SslContextFactory.config] - No Client EndPointIdentificationAlgorithm configured for Client@24875bf8[provider=null,keyStore=null
,trustStore=null]
```

**Comment by mlobstein** [comment]
This is not really a bug. The vizio binding was changed to create its own httpClient that uses a SslContextFactory which has the trustAll flag set to true. This allows TV's self signed cert to be accepted and as a result Jetty's SslContextFactory code generates the warning. I looked at some of the other bindings that use this same pattern (daikin, lgwebos, somfytahoma, venstar, etc.) and there seems to be no facilities in place way to suppress the warning in those implementations.

![image](https://user-images.githubusercontent.com/26532542/228376237-b37d2401-0804-4675-a7cd-09eb3009eb2e.png)
